wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -371,8 +389,15 @@ static function save_gallery() {
 		update_post_meta( $post_id, 'order', $ids );
 		update_post_meta( $post_id, 'captions', $captions );
 
+		$settings = filter_var_array( $settings, array(
+			'columns' => FILTER_VALIDATE_INT,
+			'link' => FILTER_SANITIZE_STRING,
+			'size' => FILTER_VALIDATE_INT,
+			'random' => FILTER_VALIDATE_BOOLEAN
+		) );
```

Comment:
This is causing this warning when I was testing:

```
Warning: filter_var_array() expects parameter 1 to be array, null given in /app/boston-chefs/externals/wp-remember-the-galleries/wp-remember-the-galleries.php on line 397
```

It still successfully saved now when I added an image to the Commonwealth gallery, but it said it failed due to the warning getting thrown and making the JSON response invalid.